client/src/pages/TourPage/TourPage.js

```diff
@@ -157,10 +145,15 @@ const TourPage = () => {
     <>
       <HeaderContent>
         <Container>
-          <Typography align="center" variant="h1" mt={17} mb={5} fontSize="50px">
+          <Typography align="center" variant="h1" mt={17} mb={3} fontSize="50px">
             {/* to be edited later with tour name from fetched data */}
             TOUR NAME
           </Typography>
+          <Stack direction="row" justifyContent="center" alignItems="center" mb={3}>
+            <Typography>{season} / </Typography>
+            <Typography>{region} / </Typography>
+            <Typography>{categories}</Typography>
```

Comment:
suggest adding routes to these categories to make them functional

Reply:
optional.  wont bother thinking about it now since we dont have any pages matching the topics